app/controllers/Application.scala

```diff
@@ -74,14 +74,18 @@ class Application @Inject() (
     }
   }
 
-  def widget(version: Option[String]) = Action { request =>
-    Ok(views.html.widget(news(version)))
+  def widget(version: Option[String]) = Action.async { request =>
+    Future.successful(
+      Ok(views.html.widget(news(version)))
+    )
   }
 
   // This used to be the download/getting-started page. We are keeping
   // the URL for SEO purposes only.
-  def download = Action { implicit request =>
-    MovedPermanently(routes.Application.gettingStarted.path)
+  def download = Action.async { implicit request =>
+    Future.successful(
+      MovedPermanently(routes.Application.gettingStarted.path)
+    )
```

Comment:
Hmm... instead of using `Action.async { ... Future.successful( ...` you could have changed the implicit request to be `implicit request: Request[AnyContent]`

Is there a reason you choose your approach?

Reply:
Ok 😉